readme.md

```diff
@@ -61,12 +61,11 @@ flatpak install com.usebruno.Bruno
 
 # On Linux via Apt
 sudo mkdir -p /etc/apt/keyrings
+sudo apt-get update && sudo apt-get install gpg
+sudo gpg --list-keys
 sudo gpg --no-default-keyring --keyring /etc/apt/keyrings/bruno.gpg --keyserver keyserver.ubuntu.com --recv-keys 9FA6017ECABE0266
-
 echo "deb [arch=amd64 signed-by=/etc/apt/keyrings/bruno.gpg] http://debian.usebruno.com/ bruno stable" | sudo tee /etc/apt/sources.list.d/bruno.list
-
-sudo apt update
-sudo apt install bruno
+sudo apt-get update && sudo apt-get install bruno
```

Comment:
I believe `apt` should be used for end-users and `apt` for scripts.

[Source](https://askubuntu.com/questions/445384/what-is-the-difference-between-apt-and-apt-get)

Reply:
Thanks for pointing out! I didn't know that - I was working with distributions without `apt` installed by default, so I wrote `apt-get` here. I changed it back to `apt` in recent commit.